### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by gwbres**
- [x] #103 
- [x] `antex` feature
- [x] improved plots x axis: use MJD 
newplot(10)
- [x] RINEX3 ionod corrections and visualization method 
- [x] Corrected all GNSS combinations
newplot(15)
newplot(18)

### CODE REVIEW

File: rinex/src/observation/record.rs [hunk 1]
```diff
@@ -1461,13 +1441,13 @@ fn dual_freq_combination(
                 };
 
                 let value = match combination {
-                    Combination::NarrowLane => alpha * beta * (v_i + gamma * v_j),
-                    _ => alpha * beta * (v_i - gamma * v_j),
+                    Combination::NarrowLane => alpha * (beta * v_i + gamma * v_j),
```
Comment (lnicola) [inline rinex/src/observation/record.rs:1444]: This is intended, right?

File: rinex/src/antex/record.rs [hunk 2]
```diff
@@ -203,25 +412,25 @@ impl Merge for Record {
...
-                                ffrequencies.push(frequency.clone());
-                            }
-                        }
-                    }
-                }
-            } else {
-                self.push(antenna.clone());
-            }
-        }
+    fn merge_mut(&mut self, _rhs: &Self) -> Result<(), merge::Error> {
```
Comment (lnicola) [inline rinex/src/antex/record.rs:415]: Is this supposed to do nothing?

**Comment by lnicola** [comment]
`rinex/src/antex/antenna/cospar.rs` is an empty file, probably not intended.

File: rinex/src/observation/record.rs (hunk 1 above)
Comment (gwbres) [reply rinex/src/observation/record.rs:1444]: Correct, the implemented equation is simplified

**Comment by gwbres** [comment]
> `rinex/src/antex/antenna/cospar.rs` is an empty file, probably not intended.

Correct, it is declared in src/antex/antenna/sv.rs, SV ou RX being the two types of antenna we can have.
Thank you for catching this

**Review by lnicola — APPROVED** [review]

File: rinex/src/antex/record.rs (hunk 2 above)
Comment (gwbres) [reply rinex/src/antex/record.rs:415]: hum, good catch once again. I forgot to re-enable this.
Reworking ATX requires to rework this method, I just forgot about it.
I will actually push one or two tests for this too.

FYI for each RINEX format, the idea is to support the Merge trait, to merge RINEX B into RINEX A, if their types matches. Basically, this is a fileops that was historically provided by teqc. 
It turns out to actually be mandatory in RINEX post processing and I use it all the time.
RINEX post processing requires putting toghethers entire datasets. One RINEX spans 24h of data. With Merge we can put a dataset together, or even cover 48h of data for example. A RINEX by itself is not enough for navigation, you need at least 3 or 4 of them.

**Pull request merged**